libkbfs/folder_block_ops.go

```diff
@@ -1863,6 +1863,15 @@ func (fbo *folderBlockOps) startSyncWrite(ctx context.Context,
 		dirtyDe = &de
 	}
 
+	// Leave a copy of the syncOp in `unrefCache`, since it may be
```

Comment:
Can you instead make a copy and pass it to `md.AddOp()` and `syncState`? That seems clearer to me.